src/components/DataPlatformProvider/DataPlatformProviderLoader.tsx

```diff
@@ -138,7 +138,9 @@ const Invalid = ({
       </PanelTitle>
     )}
   >
-    {`The data platform provider ${provider} is not recognized.`}
+    <div className="quiet">
+      {`The data platform provider “${provider}” is not recognized.`}
```

Comment:
Curly quotes? <shakes head> and I thought I knew you...~

Reply:
😱 Curly quotes forever! (I am as much of a typographical snob as I can afford to be.)